test/cloudwatchlogs/publish_logs_test.go

```diff
@@ -86,8 +108,28 @@ type cloudWatchLogGroupClassTestInput struct {
 	logGroupClass types.LogGroupClass
 }
 
+type expectedEntity struct {
+	entityType   string
+	name         string
+	environment  string
+	platformType string
+	instanceId   string
+}
+
 func init() {
 	environment.RegisterEnvironmentMetaDataFlags()
+	awsCfg, err := config.LoadDefaultConfig(
+		context.Background(),
+		config.WithRegion(pdxRegionalCode),
+	)
+	if err != nil {
+		log.Fatalf("Failed to load default config: %v", err)
+	}
+
+	cwlClient = cloudwatchlogs.NewFromConfig(awsCfg, func(o *cloudwatchlogs.Options) {
```

Comment:
we don't need to create a new client. we have globally available client that already uses us-west-2 as the region" https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/util/awsservice/constant.go#L74